analyzers/tests/SonarAnalyzer.Test/Rules/CallerInformationParametersShouldBeLastTest.cs

```diff
@@ -44,7 +44,8 @@ public class CallerInformationParametersShouldBeLastTest
 
         [TestMethod]
         public void CallerInformationParametersShouldBeLast_CSharp11() =>
-            builder.AddPaths("CallerInformationParametersShouldBeLast.CSharp11.cs").WithOptions(ParseOptionsHelper.FromCSharp11).Verify();
+            builder.AddPaths("CallerInformationParametersShouldBeLast.CSharp11.cs").WithOptions(ParseOptionsHelper.FromCSharp11)
+                .VerifyNoIssues();   // overriding an abstract default implementation is compliant
```

Comment:
We don't need this, it's not a FN
```suggestion
                .VerifyNoIssues();
```

Reply:
From just reading the test it is surprising that it is not expected to raise issues, which is why I added the comment.